e2e/cypress/integration/enterprise/cloud/billing/payment_information_after_subscription_spec.js

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+const dayJs = require('dayjs');
+
+describe('System Console - Payment Information section', () => {
+    const currentDate = new Date();
+
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        // # Visit Subscription page
+        cy.visit('/admin_console/billing/subscription');
+
+        // * Check for Subscription header
+        cy.contains('.admin-console__header', 'Subscription').should('be.visible');
+
+        // # Click Subscribe Now button
+        cy.contains('span', 'Upgrade Now').parent().click();
```

Comment:
Thanks a lot @vinod-demansol . I am getting this failure locally (see image below).
<img width="1792" alt="Screenshot 2021-09-08 at 11 52 12" src="https://user-images.githubusercontent.com/28563179/132478457-ec0a9374-df03-4d10-aad7-e8f3eba6c810.png">

Reply:
corrected the caption of subscribe button, please rereview it